_updates/2021-04-19-april.md

```diff
@@ -0,0 +1,68 @@
+---
+title: April 2021
+description: sensenet frontend updates of April 2021
+---
+
+# April 2021 🌸
+
+"The Romans gave this month the Latin name Aprilis but the derivation of this name is uncertain. The traditional etymology is from the verb aperire, "to open", in allusion to its being the season when trees and flowers begin to "open", which is supported by comparison with the modern Greek use of άνοιξη (ánixi) (opening) for spring." /from Wikipedia/ 🌷💐
+
+For us not only the flowers and trees are opening, but the second quarter in this year is also started. We have so many plans for the Q2, we are really excited to share with you soon. We would like to renew our example apps and we also would like to add some new ones - the work has already started on this. We also changed the whole build process, so you have more options for using our packages now. 🔧
+
+-----------
+
+## admin-ui@1.19.0
+
+We were also busy with fixing bugs in the last couple of weeks, e.g.: 🐛
+
+[#1251](https://github.com/SenseNet/sn-client/pull/1251) ⏰
+The end2end tests made it clear to us that a lot depends on the timing, so we added a loader to the 'Add button list' till it loads the new list.
+
+[#1252](https://github.com/SenseNet/sn-client/pull/1252) 🚩
+We cleaned up the icons also, it is now more flexible, then ever before. You can select the icon what you would like to use from the CTD itself.
+
+[#1279](https://github.com/SenseNet/sn-client/pull/1279) ♻️
+Another main point what we are working on is the webhooks. We would like to create webhook templates to make your life easier with workflows.
+Keep up with the next releases, it will be available soon, the first step for this was the editable payload.
+
+... and many more [#1248](https://github.com/SenseNet/sn-client/pull/1248), [#1250](https://github.com/SenseNet/sn-client/pull/1250).
+
+
+## @sensenet/document-viewer-react@4.1.0
+
+In document viewer we had also some bugs or missing features what we tried to solve:
+
+[#1259](https://github.com/SenseNet/sn-client/pull/1259) 📑
+Comment marker placing on inactive page has a bug, now it is working properly.
+
+[#1256](https://github.com/SenseNet/sn-client/pull/1256) 🔄
+One, not a very likely case is handled: if the backend cannot generate previews for a long time, it will not affect other opreations, infinite loop cannot be created.
```

Comment:
```suggestion
One edgecase is also has been adressed: if the backend cannot generate previews for a long time, it will not affect other opreations, infinite loop cannot be occured.
```